indico/modules/events/contributions/util.py

```diff
@@ -226,6 +226,9 @@ def import_contributions_from_csv(event, f):
         if not title:
             raise UserValueError(_("Row {}: contribution title is required").format(num_row))
 
+        if not validate_email(email):
```

Comment:
Still not the correct logic. The email address is optional, but if it's set it must be valid. So you need `if email and not validate_email(email)` here (this is actually why the tests are currently failing).